### ISSUE TITLE
TypeError: Cannot read property 'prototype' of undefined","    at __extends

### ISSUE THREAD

**Issue by ekoome** [bjyoungblood/es6-error#28]
Hi 

Getting above error when using with Typescript 2.1. This is where the error occurs on ES5 code generated under b.prototype:

`var __extends = (this && this.__extends) || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
};`

Here's is how i'm calling the module:

`import` ExtendableError  from "es6-error";

export class AuthenticationError extends ExtendableError {}`

The ES5 version works:

`var util = require('util');
var ExtendableError = require('es6-error');

function AuthenticationError(message) {
  message = message || 'Default message';
  ExtendableError.call(this, message);
}

util.inherits(AuthenticationError, ExtendableError);

export {AuthenticationError}`

What am i doing wrong?

**Comment by bjyoungblood** [comment]
It looks like this is happening because of our use of `babel-plugin-add-module-exports` in the CommonJS build. I published `v4.0.2` to address this.

In order to use the correct build, you will need to add add this alias to `resolve.alias` in your webpack config:

```
{
  'es6-error' : 'es6-error/lib/index.ts.js'
}
```

**Comment by ekoome** [comment]
The error I'm getting is runtime rather than build time. I'm not using webpack rather typescript- tsconfig.

**Comment by bjyoungblood** [comment]
I'm not very familiar with TypeScript, but I think you can achieve the same thing by adding this to your `tsconfig.json`:

```
{
  "compilerOptions": {
    "baseUrl": ".", // This must be specified if "paths" is.
    "paths": {
      "es6-error": ["node_modules/es6-error/lib/index.ts.js"]
    }
  }
}
```

**Comment by ekoome** [comment]
Is it easier to standardise this to established convention  to have node_modules/es6-error/index.ts or index.d.ts

**Comment by dyihoon90** [comment]
Hi, I am sadly still running into this issue in 4.0.2, would you mind taking a look at my PR
#32 I changed the index.d.ts file to conform to module.exports

**Comment by bjyoungblood** [comment]
I think this should be resolved by #36. Please reopen if that is not the case.
